pallets/api/src/fungibles/tests.rs

```diff
@@ -147,6 +147,30 @@ fn clear_metadata_works() {
 	});
 }
 
+#[test]
+fn mint_works() {
```

Comment:
Should we cover more cases like `Asset` does not exist?

Reply:
Like I've mentioned before, because it is a 1:1 mapping to the pallet assets dispatchable we should have to test the functionality, only if it works or not. Only when adding logic the test should be expanded (e.g. `approve`).